q2_metadata/_random.py

```diff
@@ -14,14 +14,24 @@
 def shuffle_groups(metadata: qiime2.CategoricalMetadataColumn,
                    n_columns: int = 3,
                    column_name_prefix: str = 'shuffled.grouping.',
-                   column_value_prefix: str = 'fake.group.') -> pd.DataFrame:
+                   # Suggest renaming this variable to sample_value_prefix for
+                   # clarity
```

Comment:
```suggestion
```
I discussed this with @cherman2 and what we ended up agreeing on was adding an `md` prefix to these two parameter names so that it's clear that it's referring to the metadata column name, and then the value names within the metadata column. I've added those two suggestions for you inline!

Reply:
Thank you! I agree :)